app/controllers/application_controller.rb

```diff
@@ -21,6 +21,21 @@ def current_user
   end
   helper_method :current_user
 
+  def current_group
+    if params[:group_id]
+      @group ||= Group.find(params[:group_id])
+    else
+      @group ||= Group.find(params[:id])
+    end
+    @group
+  end
```

Comment:
are you sure about having this kind of global `current_group` helper which falls back to find by id (whatever resource/page are you on?)? And please try to use strong parameters :) and since this changes quite a lot in Rails 5.0 (type of `params` changes from `Hash` to `ActiveController::Parameters`, i advise to upgrade to 5.0 then 5.1 if it's possible. 

besides the semantic issue, if you need a construct like this, there is a more elegant way in ruby:

```ruby
@group = if condition
                   Group.find(...)
                  else
                    # something else
                  end 
```